pyproject.toml

```diff
@@ -8,6 +8,7 @@ authors = []
 python = "^3.7"
 flask = "^1.1.2"
 python-dotenv = "^0.14.0"
+requests = "^2.26.0"
```

Comment:
I'd expect to see `pytest` added as a dependency to this project (optionally as development only, which you can do with `poetry add -D pytest`)